### PR TITLE
Wishlist/ Cart management

### PR DESCRIPTION
**Pull request by varun-Tad**
Created Wishlist page & cart management Page.Please review.
You can navigate to both the pages from the landing page

### CODE REVIEW

File: Pages/Cart-page/cart.css [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+
+
+.check-container .total{
+    display: flex;
+    justify-content: space-between;
+    border-top: 1px solid black;
+    border-bottom: 1px solid black;
+    padding:1rem 0 1rem 1rem;
+    margin-bottom: 1rem;
+}
```
Comment (rdthetha) [inline Pages/Cart-page/cart.css]: make a variable for black color as it used multiple times

**Comment by amantulsyan35** [comment]
The html structure looks good to me, the CSS naming can be improved I guess, "entire-card" is a bit vague something like cart-card or checkout-card or  checkout-product-card is more descriptive for the person reading the code when the app scales

**Comment by varun-Tad** [comment]
Okay, will work on it.Thanks @amantulsyan35

File: Pages/Cart-page/cart.css (hunk 1 above)
Comment (varun-Tad) [reply Pages/Cart-page/cart.css]: Okay. will make the required changes.Thanks @rdthetha

**Pull request merged**